packages/core/src/baseclient.ts

```diff
@@ -392,9 +392,11 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
       return SyncPromise.reject('SDK not enabled, will not send event.');
     }
 
+    const isTransaction = event.type === 'transaction';
     // 1.0 === 100% events are sent
     // 0.0 === 0% events are sent
-    if (typeof sampleRate === 'number' && Math.random() > sampleRate) {
+    // Sampling for transaction happens somewhere else
+    if (!isTransaction && typeof sampleRate === 'number' && Math.random() > sampleRate) {
```

Comment:
We should either use `Math.random() > sampleRate` or `Math.random() < sampleRate`, not one here and one there (in transaction)

Reply:
But it works different, `sampled` true in tracing means we want to have it.
Added a comment:
```
      // if true = we want to have the transaction
      // if false = we don't want to have it
      // Math.random (inclusive of 0, but not 1)
      transaction.sampled = Math.random() < sampleRate;
```

Reply:
![image](https://user-images.githubusercontent.com/1523305/82895749-fbdf3500-9f54-11ea-9974-3f3249be2e49.png)